### PR TITLE
replace exports with module.exports

### PR DESCRIPTION
**Pull request by vaishnav98**
I have made the suggested changes. can you please review it.I did verify the exported objects before and after the change was made.
Issue #82

### CODE REVIEW

File: src/hw_mainline.js [hunk 1]
```diff
@@ -370,4 +370,23 @@ exports.readPlatform = function (platform) {
...
+    exportGPIOControls: exportGPIOControls,
+    writeGPIOValue: writeGPIOValue,
+    readGPIOValue: readGPIOValue,
+    enableAIN: enableAIN,
+    readAIN: readAIN,
+    writeGPIOEdge: writeGPIOEdge,
+    writePWMFreqAndValue: writePWMFreqAndValue,
+    readEeproms: readEeproms,
+    readPlatform: readPlatform,
+
```
Comment (jadonk) [inline src/hw_mainline.js]: Looks extraneous.

File: src/hw_oldkernel.js [hunk 2]
```diff
@@ -198,6 +198,25 @@ exports.readEeproms = function (eeproms) {
...
+    exportGPIOControls: exportGPIOControls,
+    writeGPIOValue: writeGPIOValue,
+    readGPIOValue: readGPIOValue,
+    enableAIN: enableAIN,
+    readAIN: readAIN,
+    writeGPIOEdge: writeGPIOEdge,
+    writePWMFreqAndValue: writePWMFreqAndValue,
+    readEeproms: readEeproms,
+    readPlatform: readPlatform,
+
```
Comment (jadonk) [inline src/hw_oldkernel.js]: Another one. Not sure why the blank lines.

File: src/hw_simulator.js [hunk 3]
```diff
@@ -104,8 +104,27 @@ exports.readEeproms = function (eeproms) {
...
+    exportGPIOControls: exportGPIOControls,
+    writeGPIOValue: writeGPIOValue,
+    readGPIOValue: readGPIOValue,
+    enableAIN: enableAIN,
+    readAIN: readAIN,
+    writeGPIOEdge: writeGPIOEdge,
+    writePWMFreqAndValue: writePWMFreqAndValue,
+    readEeproms: readEeproms,
+    readPlatform: readPlatform,
+
```
Comment (jadonk) [inline src/hw_simulator.js]: Again, extraneous. The repetition makes me wonder if the definition of what gets exported from these hw_X.js files should be common as they satisfy an internal API.

File: src/hw_universal.js [hunk 4]
```diff
@@ -293,4 +293,23 @@ exports.readPlatform = function (platform) {
...
+    exportGPIOControls: exportGPIOControls,
+    writeGPIOValue: writeGPIOValue,
+    readGPIOValue: readGPIOValue,
+    enableAIN: enableAIN,
+    readAIN: readAIN,
+    writeGPIOEdge: writeGPIOEdge,
+    writePWMFreqAndValue: writePWMFreqAndValue,
+    readEeproms: readEeproms,
+    readPlatform: readPlatform,
+
```
Comment (jadonk) [inline src/hw_universal.js]: Another one.

File: src/index.js [hunk 5]
```diff
@@ -667,9 +665,10 @@ process.nextTick(run);
 // Global variable assignments
 // This section is broken out because it will eventually be deprecated
 function setGlobals() {
```
Comment (jadonk) [inline src/index.js]: Given the way things are done, this could probably be f.setGlobals = function() {.

File: src/rc.js [hunk 6]
```diff
@@ -68,4 +68,14 @@ exports.rcEncoder = function (encoder, value, callback) {
...
+rcEncoder.args = ['encoder', 'value', 'callback'];
+
+module.exports = {
+    rcInitialize: rcInitialize,
+    rcState: rcState,
+    rcLED: rcLED,
+    rcOn: rcOn,
+    rcMotor: rcMotor,
+    rcEncoder: rcEncoder
+
```
Comment (jadonk) [inline src/rc.js]: Another blank line. Inconsistent on ending with commas or not.

File: src/index.js (hunk 5 above)
Comment (vaishnav98) [reply src/index.js]: Yes, that would be the better way , i will make the correction.

File: src/hw_mainline.js (hunk 1 above)
Comment (vaishnav98) [reply src/hw_mainline.js]: Sorry i did not notice this , i copied the exports for the hw_X.js as the exports are same for these files, that was why the mistake was repeated, i will be more careful in future.

**Comment by vaishnav98** [comment]
https://github.com/beagleboard/bonescript/issues/24

**Pull request merged**